packages/core/test/uiSchema_test.js

```diff
@@ -1201,11 +1204,11 @@ describe("uiSchema", () => {
           schema,
           uiSchema,
           formData: {
-            foo: 3.14,
+            foo: 13.14,
```

Comment:
The input range has a minimum of 10. The fact that JSDOM let you set 3.14 before was faulty, so I increased the value to within the valid input range.